### PR TITLE
BACKLOG-10271 : use and bind table component in content picker.

### PR DESCRIPTION
**Pull request by dgriffon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
         "@material-ui/icons": "^3.0.1",
         "@material-ui/lab": "^3.0.0-alpha.30",
         "ckeditor4-react": "^0.1.0",
+        "dayjs": "^1.8.14",
```
Comment (FBerthelot) [inline package.json]: <3

File: src/javascript/DesignSystem/ContentTable/ContentTable.jsx [hunk 2]
```diff
@@ -27,10 +27,37 @@ const styles = theme => ({
         '&:hover': {
             backgroundColor: theme.palette.background.default
         }
+    },
+    selectedRow: {
+        '&&&': {
```
Comment (FBerthelot) [inline src/javascript/DesignSystem/ContentTable/ContentTable.jsx]: Why 3 & ?

I don't know this selector

File: src/javascript/DesignSystem/ContentTable/ContentTableHeader/ContentTableHeader.jsx [hunk 3]
```diff
@@ -11,7 +10,7 @@ const ContentTableHeader = ({columns, order, orderBy}) => {
         <TableHead>
             <TableRow>
                 <TableCell padding="checkbox">
-                    <Checkbox indeterminate/>
+                    {/* Checkbox location */}
```
Comment (FBerthelot) [inline src/javascript/DesignSystem/ContentTable/ContentTableHeader/ContentTableHeader.jsx]: Why removing this element ? Maybe add a TODO ?

File: src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/ContentPicker/ContentPickerEmpty/ContentTable/ContentTable.container.jsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import DSContentTable from '../../../../../../../../DesignSystem/ContentTable/ContentTable';
+import {useQuery} from 'react-apollo-hooks';
+import {translate} from 'react-i18next';
+import {ProgressOverlay} from '@jahia/react-material';
+import * as PropTypes from 'prop-types';
```
Comment (FBerthelot) [inline src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/ContentPicker/ContentPickerEmpty/ContentTable/ContentTable.container.jsx]: Avoid import * syntax.

```suggestion
import PropTypes from 'prop-types';
```

is better

File: src/javascript/DesignSystem/ContentTable/ContentTable.jsx (hunk 2 above)
Comment (dgriffon) [reply src/javascript/DesignSystem/ContentTable/ContentTable.jsx]: This came from the CMM table styles, in any case the table will be redesign

File: src/javascript/DesignSystem/ContentTable/ContentTableHeader/ContentTableHeader.jsx (hunk 3 above)
Comment (dgriffon) [reply src/javascript/DesignSystem/ContentTable/ContentTableHeader/ContentTableHeader.jsx]: yes, a TODO is better, thank you

File: src/javascript/DesignSystem/ContentTable/ContentTable.jsx (hunk 2 above)
Comment (FBerthelot) [reply src/javascript/DesignSystem/ContentTable/ContentTable.jsx]: oki :)

**Pull request merged**